server/common/app_config.py

```diff
@@ -35,6 +35,15 @@ def __init__(self, **kw):
         self.enable_reembedding = False
         self.anndata_backed = False
 
+        # The index page when in multi-dataset mode:
+        #   False or None:  this returns a 404 code
+        #   True:  loads a test index page, which links to the datasets that are available in the dataroot
+        #   string/URL:  redirect to this URL:  flask.redirect(config.multi_dataset_index)
+        self.multi_dataset_index = True
```

Comment:
I'd prefer this were off by default - is there any value in having debugging route on by default?

Reply:
Okay, I can turn that off.  I was already off in the eb app.